npm/vue/cypress/plugins/index.js

```diff
@@ -5,7 +5,11 @@ const webpackConfig = require('../../webpack.config')
 /**
  * @type Cypress.PluginConfig
  */
-module.exports = (on, config) => {
+module.exports = (on, config, mode) => {
+  if (mode !== 'component') {
+    throw Error('This is a component testing project. mode should be `component`.')
```

Comment:
Maybe give a hint to users on what they did wrong:

`Please use open-ct instead of open for this project`.

Reply:
This is in our tests, not in the user code